File: RewardSampleForObjC/RewardSampleForObjC/UIColor+Extension.m
//
//  UIColor+Extension.m
//  RewardSampleForObjC
//
//  Created by AdGeneration on 2019/09/10.
//  Copyright © 2019 Supership Inc. All rights reserved.
//

#import "UIColor+Extension.h"

@implementation UIColor (Extension)
+ (UIColor *)defaultLabelColor {
    if (@available(iOS 13.0, *)) {
        return UIColor.labelColor;
    } else {
        return UIColor.blackColor;
    }
}
@end

File: RewardSampleForSwift/RewardSampleForSwift/UIColor+Extension.swift
//
//  UIColor+Extension.swift
//  RewardSampleForSwift
//
//  Created by AdGeneration on 2019/09/10.
//  Copyright © 2019 Supership Inc. All rights reserved.
//

import Foundation
import UIKit

extension UIColor {
    public class func defaultLabelColor() -> UIColor {
        if #available(iOS 13.0, *) {
            return UIColor.label
        } else {
            return UIColor.black
        }
    }
}
